include/seqan3/alignment/pairwise/detail/policy_affine_gap_recursion.hpp

```diff
@@ -214,8 +214,8 @@ class policy_affine_gap_recursion
      */
     score_type lowest_viable_score() const noexcept
     {
-        assert(m_gap_open_score <= 0 && m_gap_extension_score <= 0);
-        return std::numeric_limits<score_type>::lowest() - (m_gap_open_score + m_gap_extension_score);
+        assert(gap_open_score <= 0 && gap_extension_score <= 0);
```

Comment:
I don't see the name change in the actual member variable in this commit ([MISC] Remove m_ prefix from affine internal variables.)?

Reply:
Probably, because of some rebasing issues.

Reply:
I will resolve the commit once I do the final rebase.